### ISSUE TITLE
[FSE] Add option to display button labels instead of icons

### ISSUE THREAD

**Issue by aurooba** [WordPress/gutenberg#38256]
## What problem does this address?
Icons aren't always the friendliest way to interact with interfaces in general, labels and texts are often better at helping people acclimate to a new interface.

## What is your proposed solution?
 I'd love if the Site Editor also included the option to `Display button labels` like the Block Editor does in some sort of `Preferences` area – and in fact, these options should be connected, so setting in the Block Editor context should be honoured in the Site Editor context, and vice versa.

<img width="359" alt="Screen Shot 2022-01-26 at 3 20 31 PM" src="https://user-images.githubusercontent.com/6925260/151256738-ae419dd7-2b96-44b7-af42-6da58e80de6e.png">

**Comment by annezazu** [comment]
This was handled in a PR that landed in Gutenberg 12.9 and it'll land in 6.0 :) 

https://github.com/WordPress/gutenberg/pull/38317

Closing this out as a result.
